### PR TITLE
Enable conditionally displaying governance connector related UI elements

### PR DESCRIPTION
**Pull request by pavinduLakshan**
### Purpose

$subject

### Related Issues
- N/A

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [X] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [X] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [X] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/server-configurations/forms/login-attempt-security-form.tsx [hunk 1]
```diff
@@ -557,25 +559,39 @@ export const LoginAttemptSecurityConfigurationFrom: FunctionComponent<
                 {
                     sampleInfoSection()
                 }
-                <Field.Checkbox
-                    ariaLabel="notifyUserOnAccountLockIncrement"
-                    name="notifyUserOnAccountLockIncrement"
-                    label={ t("extensions:manage.serverConfigurations.accountSecurity." +
+
```
Comment (Achintha444) [inline apps/console/src/features/server-configurations/forms/login-attempt-security-form.tsx:562]:
```suggestion
```

File: apps/console/src/features/server-configurations/forms/login-attempt-security-form.tsx [hunk 2]
```diff
@@ -557,25 +559,39 @@ export const LoginAttemptSecurityConfigurationFrom: FunctionComponent<
...
+                                t("extensions:manage.serverConfigurations.accountSecurity." +
                             "loginAttemptSecurity.form.fields.notifyUserOnAccountLockIncrement.hint")
-                    }
-                </Hint>
+                            }
+                        </Hint>
+                    )
+                }
+
+
```
Comment (Achintha444) [inline apps/console/src/features/server-configurations/forms/login-attempt-security-form.tsx:594]:
```suggestion
```

Comment (pavinduLakshan) [reply apps/console/src/features/server-configurations/forms/login-attempt-security-form.tsx]: Fixed with c9a9513

File: apps/console/src/features/server-configurations/forms/login-attempt-security-form.tsx (hunk 1 above)
Comment (pavinduLakshan) [reply apps/console/src/features/server-configurations/forms/login-attempt-security-form.tsx]: Fixed with c9a9513

**Review by Achintha444 — APPROVED** [review]

**Pull request merged**